### DISCUSSION TITLE
Allow Project-Specific Default Time Ranges

### DISCUSSION THREAD

**Discussion by dkindlund** [langfuse/langfuse#1305] [Ideas]
### Describe the feature or potential improvement

When a user logs into their Langfuse project, it would be awesome if they could set a default timespan that would remain "sticky", rather than having to change it every time they revisit the page:

<img width="478" alt="image" src="https://github.com/langfuse/langfuse/assets/85660/5117677b-ba69-4ab4-8c35-7621c5234ffb">

**Comment by marcklingen** [comment]
Thanks for raising this. We could just persist it in local storage as defaulting to the last used time range is probably a good idea.
